test/ctdbp_no_calibration_coefficients.cql

```diff
@@ -1,4 +1,4 @@
-CREATE TABLE ctdbp_no_calibration_coefficients (
+CREATE TABLE stream_engine_test.ctdbp_no_calibration_coefficients (
```

Comment:
I think this behavior must depend on the version of cassandra... I ran into this on one machine but not the other. Thanks for fixing...